index.html

```diff
@@ -172,38 +173,45 @@ <h2>Past Meetups</h2>
+            hang out together, chat, listen to talks, exchange ideas and make new friends.
+            Any harmful or discriminating behaviour will not be tolerated and results
+            in the offending person being expelled from the meetup.
+          </p>
+          <p>
+            For details on what kinds of behaviour are not tolerated and consequences for violating these rules, we prefer to the <a href="http://rubyberlin.github.io/code-of-conduct">Berlin Code of Conduct</a>.
+          </p>
+        </div>
+      </section>
       <section>
         <div class="clearfix one-col">
-          <div>
-            <h2>Code of Conduct</h2>
-            <p>
-              Our goal is to have an awesome, inclusive and safe community meetup where people meet,
-              hang out together, chat, listen to talks, exchange ideas and make new friends.
-              Any harmful or discriminating behaviour will not be tolerated and results
-              in the offending person being expelled from the meetup.
-            </p>
-            <p>
-              For details on what kinds of behaviour are not tolerated and consequences for violating these rules, we refer to the <a href="http://rubyberlin.github.io/code-of-conduct">Berlin Code of Conduct</a>.
-            </p>
-          </div>
           <div>
             <h2>Imprint, privacy policy and legal information</h2>
             <p>
               This site is run by
               <a href="http://bitcrowd.net" title="bitcrowd.net software development for startups">bitcrowd</a> -
-              please find <a href="http://bitcrowd.net/legal" title="bitcrowd.net software development legal information">imprint, privacy and other legal information here</a>.
+              please find <a href="http://bitcrowd.net/legal"
+                title="bitcrowd.net software development legal information">imprint, privacy and other legal information
+                here</a>.
             </p>
           </div>
         </div>
       </section>
     </footer>
+
     <script>
-      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
-      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
-      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-      })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
+      (function (i, s, o, g, r, a, m) {
+      i['GoogleAnalyticsObject'] = r; i[r] = i[r] || function () {
+        (i[r].q = i[r].q || []).push(arguments)
+      }, i[r].l = 1 * new Date(); a = s.createElement(o),
+        m = s.getElementsByTagName(o)[0]; a.async = 1; a.src = g; m.parentNode.insertBefore(a, m)
+      })(window, document, 'script', 'https://www.google-analytics.com/analytics.js', 'ga');
       ga('create', 'UA-87838740-1', 'auto');
       ga('send', 'pageview');
     </script>
   </body>
-</html>
+</html>
```

Comment:
I think maybe your editor stripped the LR at the end of the file (that’s what the little red icon means)? For consistency, all files should have a line-break at the end (your editor can be setup to do that automatically :) )

Reply:
Fixed and we changed the settings so that from now on it adds the LR in it's own, thanks! ✨